packages/desktop/Cargo.toml

```diff
@@ -32,6 +32,7 @@ dioxus-html = { path = "../html", features = ["serialize"], version = "^0.1.6" }
 webbrowser = "0.5.5"
 mime_guess = "2.0.3"
 dioxus-interpreter-js = { path = "../interpreter", version = "^0.0.0" }
+image = "0.24.0"
```

Comment:
Is there a way to implement this feature without relying on the `image` crate?

Can we do the version once and then just load the converted bytes in at runtime?

I ask because image takes up some (quite a bit?)x compile time.

Reply:
we can dump the bytes